packages/spaces/commands/create.js

```diff
@@ -81,7 +81,7 @@ module.exports = {
     { name: 'shield', hasValue: false, hidden: true, description: 'create a Shield space' },
     { name: 'cidr', hasValue: true, description: 'the RFC-1918 CIDR the space will use' },
     { name: 'kpi-url', hasValue: true, hidden: true, description: 'self-managed KPI endpoint to use' },
-    { name: 'data-cidr', hasValue: true, description: "the RFC-1918 CIDR that the space will use when peering with DoD's spaces" },
+    { name: 'data-cidr', hasValue: true, description: "the RFC-1918 CIDR used for Heroku Data resources for the space" },
```

Comment:
"for...for" sounds a little odd to me, and I feel the "the" is unnecessary. Maybe, "RFC-1918 CIDR used to address Heroku Data resources in the space"? I'm not even sure if that's a great description though since it isn't clear what this does. In any case I think this could be clearer.

Reply:
also, CI is failing here because this should use single quotes

Reply:
@jdxcode tweaked the wording and fixed quotes ptal